testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -129,24 +129,42 @@ def selector_params(module_label):
 @pytest.fixture(scope="module")
 def authorino_labels(selector_params) -> Dict[str, str]:
     """Labels for the proper Authorino discovery"""
-    labels = {
-        "authorino.kuadrant.io/managed-by": "authorino",
-        selector_params[0]: selector_params[1]
-    }
+    labels = {"authorino.kuadrant.io/managed-by": "authorino", selector_params[0]: selector_params[1]}
     return labels
 
 
 # pylint: disable-msg=too-many-locals
 @pytest.fixture(scope="module")
-def envoy(request, authorino, openshift, create_secret, blame, label, backend,
-          authorino_authority, envoy_authority, envoy_cert, testconfig, authorino_labels):
+def envoy(
```

Comment:
This type of change is nightmare for every sane person.

Reply:
not at right enough?
interesting that it was "fine" here: ;-)
https://github.com/Kuadrant/testsuite/pull/145/files#diff-e242f4968001181ce82739996a5e823dc762bb2e988c8bf27d1c6e24240f7126L62